readme.md

```diff
@@ -1,7 +1,43 @@
+# EarTrainingApp
+
+## Presentation
+
+This repository is open for contributions.
+You can see the demo of the app here: https://chord-progression-ear-training.herokuapp.com/
+
+The app has actually two modes, a chord-progression game and a chord-and-note game.
+The chord-and-note game let you see your progress day-by-day if you login.
+<br />
+
+## How to run it locally
+
+clone the repo.<br />
+setup a mongodb database locally.<br />
+setup the environment variables.<br />
+start the database `mongod --dbpath ~/[path to your database]`<br />
+`npm install`<br />
+`npm start`<br />
+visit http://localhost:8080
+<br />
+
+### Environment Variables Setup
+
+The app uses environment variable to point to the database.
+You can create a mongo db anywhere you want on your computer, just change the path accordingly when you start the database,
```

Comment:
just a quick note
env vars are totally legit way of manipulating configs in apps nowadays
let users deside how to load their values to the env vars
it can be inside the run command (in the same line), having it on the dev machine, or by injecting them into the docker container on production...